arlo-client/wdio.conf.js

```diff
@@ -0,0 +1,280 @@
+  // Services take over a specific job you don't want to take care of. They enhance
+  // your test setup with almost no effort. Unlike plugins, they don't add new
+  // commands. Instead, they hook themselves up into the test process.
+  services: ['chromedriver'],
+
+  // Framework you want to run your specs with.
+  // The following are supported: Mocha, Jasmine, and Cucumber
+  // see also: https://webdriver.io/docs/frameworks.html
+  //
+  // Make sure you have the wdio adapter package for the specific framework installed
+  // before running any tests.
+  framework: 'jasmine',
+  //
+  // The number of times to retry the entire specfile when it fails as a whole
+  // specFileRetries: 1,
+  //
+  // Test reporter for stdout.
+  // The only one supported by default is 'dot'
+  // see also: https://webdriver.io/docs/dot-reporter.html
+  reporters: ['dot'],
+
+  //
+  // Options to be passed to Jasmine.
+  jasmineNodeOpts: {
+    //
+    // Jasmine default timeout
+    defaultTimeoutInterval: 60000,
+    //
+    // The Jasmine framework allows interception of each assertion in order to log the state of the application
+    // or website depending on the result. For example, it is pretty handy to take a screenshot every time
+    // an assertion fails.
+    // expectationResultHandler: function(passed, assertion) {
+    //   do something
+    // },
+  },
+
+  //
+  // =====
+  // Hooks
+  // =====
+  // WebdriverIO provides several hooks you can use to interfere with the test process in order to enhance
+  // it and to build services around it. You can either apply a single function or an array of
+  // methods to it. If one of them returns with a promise, WebdriverIO will wait until that promise got
+  // resolved to continue.
+  /**
+   * Gets executed once before all workers get launched.
+   * @param {Object} config wdio configuration object
+   * @param {Array.<Object>} capabilities list of capabilities details
+   */
+  // onPrepare: function (config, capabilities) {
+  // },
+  /**
+   * Gets executed just before initialising the webdriver session and test framework. It allows you
+   * to manipulate configurations depending on the capability or spec.
+   * @param {Object} config wdio configuration object
+   * @param {Array.<Object>} capabilities list of capabilities details
+   * @param {Array.<String>} specs List of spec file paths that are to be run
+   */
+  // beforeSession: function(config, capabilities, specs) {
+  // },
+  /**
+   * Gets executed before test execution begins. At this point you can access to all global
+   * variables like `browser`. It is the perfect place to define custom commands.
+   * @param {Array.<Object>} capabilities list of capabilities details
+   * @param {Array.<String>} specs List of spec file paths that are to be run
+   */
+  // before: function (capabilities, specs) {
+  // },
+  before: function() {
+    require('ts-node').register({ files: true })
+    require('@babel/register')({
```

Comment:
I know we went over this in a previous iteration of this PR, but I still can’t remember why both `ts-node` and `@babel/register` are required. Can you add a comment?

Reply:
Before, whenever I removed `ts-node` TS would stop working in the wdio tests. I tried it again and they seem to run fine now, so I've removed it.

Reply:
@eventualbuddha Hmm perhaps I spoke too soon. Tests are now failing because TS stopped working:

```
ERROR @wdio/runner: SyntaxError: /home/circleci/project/arlo-client/src/test/specs/auditFlow.ts: Unexpected token, expected "," (10:26)

   8 | import { voteValue } from '../../components/AuditFlow/BlockRadio'
   9 | 
> 10 | const ballotNext = (option: voteValue) => {
     |                                              ^
```

Any idea why it might have worked fine right after I removed the line, but won't work now on CI (or locally)?